### DISCUSSION TITLE
Using Neper -V to generate structured 3D output from FEPX results?

### DISCUSSION THREAD

**Discussion by xshang93** [neperfepx/neper#496] [Q&A] [answered]
Hi, this is a long shoot but is it possible to use Neper to generate, say, a 32x32x32 matrix containing the von Mises stress value at each corresponding location? Basically to output the results as if the structure is mapped to have 32x32x32 cubic meshes.

Thanks

**Comment by rquey** [comment, marked as answer]
We plan to support this in -S, but, for now, you'll have to write the list of points as a [point/position file](https://neper.info/doc/fileformat.html?highlight=point%20file#position-file), load the file in `-M` using [-loadpoint](https://neper.info/doc/neper_m.html#cmdoption-loadpoint), and ask for the elements to which they belong using [-statpoint elt](https://neper.info/doc/neper_t.html?highlight=statpoint#cmdoption-statpoint).  From the list of elements, you can easily determine the point values.

**Reply by xshang93** [reply]
Sorry for the delay in reply. I'm trying out this method using the line below
`neper -M -loadmesh ./1_sim/simulation.msh -loadpoint ./1_sim.sim/results/nodes/coo/coo.step2 -statpoint elt`
but it freezes at the last step, and neper is consuming 100% of my CPU.
```
========================    N   e   p   e   r    =======================
Info   : A software package for polycrystal generation and meshing.
Info   : Version 4.4.2-33
Info   : Built with: gsl|muparser|opengjk|openmp|nlopt|libscotch (full)
Info   : Running on 4 threads.
Info   : <https://neper.info>
Info   : Copyright (C) 2003-2022, and GNU GPL'd, by Romain Quey.
Info   : No initialization file found (`/home/xiao/.neperrc').
Info   : ---------------------------------------------------------------
Info   : MODULE  -M loaded with arguments:
Info   : [ini file] (none)
Info   : [com line] -loadmesh ./1_sim/simulation.msh -loadpoint
         ./1_sim.sim/results/nodes/coo/coo.step2 -statpoint elt
Info   : ---------------------------------------------------------------
Info   : Reading input data...
Info   :   - Reading arguments...
Info   : Loading input data...
Info   :   - Loading points...
Info   :     [i] Parsing file `./1_sim.sim/results/nodes/coo/coo.step2'...
Info   :     [i] Parsed file `./1_sim.sim/results/nodes/coo/coo.step2'.
Info   : Loading result mesh...
Info   :     [i] Parsing file `./1_sim/simulation.msh'...
Info   :     [i] Parsed file `./1_sim/simulation.msh'.
Info   :   - Reconstructing mesh...
Info   : Searching nsets and fasets...
Info   : Writing mesh results...
Info   :   - Preparing mesh...
Info   :   - Mesh properties:
Info   :     > Node number:    31921
Info   :     > Elt  number:    21629
Info   :     > Mesh volume:    1.000
Info   :   - Writing mesh...
Info   :     [o] Writing file `./1_sim/simulation.msh'...
Info   :     [o] Wrote file `./1_sim/simulation.msh'.
Info   : Writing mesh statistics...
Info   :     [o] Writing file `./1_sim/simulation.stpoint'...
```
I'm on ubuntu 20.04. Is there anything I'm doing wrong?

**Reply by xshang93** [reply]
I found what went wrong:
1. I should generate my own points file based on my 3D image structures
2. I should be more patient... turns out it's not frozen but just taking some time to finish
